aptos-move/framework/aptos-framework/doc/native_bridge.md

```diff
@@ -628,6 +736,77 @@ An event triggered upon change of bridgefee
 
 
 
+<a id="0x1_native_bridge_initialize"></a>
+
+## Function `initialize`
+
+Initializes the module and stores the <code>EventHandle</code>s in the resource.
+
+
+<pre><code><b>public</b> <b>fun</b> <a href="native_bridge.md#0x1_native_bridge_initialize">initialize</a>(aptos_framework: &<a href="../../aptos-stdlib/../move-stdlib/doc/signer.md#0x1_signer">signer</a>)
+</code></pre>
+
+
+
+<details>
+<summary>Implementation</summary>
+
+
+<pre><code><b>public</b> <b>fun</b> <a href="native_bridge.md#0x1_native_bridge_initialize">initialize</a>(aptos_framework: &<a href="../../aptos-stdlib/../move-stdlib/doc/signer.md#0x1_signer">signer</a>) {
+    <a href="system_addresses.md#0x1_system_addresses_assert_aptos_framework">system_addresses::assert_aptos_framework</a>(aptos_framework);
+
+    <b>let</b> bridge_config = <a href="native_bridge.md#0x1_native_bridge_BridgeConfig">BridgeConfig</a> {
+        bridge_relayer: <a href="../../aptos-stdlib/../move-stdlib/doc/signer.md#0x1_signer_address_of">signer::address_of</a>(aptos_framework),
+        insurance_fund: <a href="../../aptos-stdlib/../move-stdlib/doc/signer.md#0x1_signer_address_of">signer::address_of</a>(aptos_framework),
+        risk_denominator: 4,
+        bridge_fee: 40_000_000_000,
+    };
+    <b>move_to</b>(aptos_framework, bridge_config);
+
+    // Ensure the nonce is not already initialized
+    <b>assert</b>!(
+        !<b>exists</b>&lt;<a href="native_bridge.md#0x1_native_bridge_Nonce">Nonce</a>&gt;(<a href="../../aptos-stdlib/../move-stdlib/doc/signer.md#0x1_signer_address_of">signer::address_of</a>(aptos_framework)),
+        2
+    );
+
+    // Create the <a href="native_bridge.md#0x1_native_bridge_Nonce">Nonce</a> resource <b>with</b> an initial value of 0
+    <b>move_to</b>&lt;<a href="native_bridge.md#0x1_native_bridge_Nonce">Nonce</a>&gt;(aptos_framework, <a href="native_bridge.md#0x1_native_bridge_Nonce">Nonce</a> {
+        value: 0
+    });
+
+    // Create the InboundRateLimitBudget resource
```

Comment:
Both inbound and outbound should be clearly defined. ideally ontop the md file. here is the text


Inbound: Transfers from the bridge to the user (e.g., tokens minted and transferred to the recipient).
Outbound: Transfers from the user to the bridge (e.g., tokens burned for bridging).

Reply:
you proposed this but i cannot find it here

// Inbound: Circulating token is released on the destination chain (unlock or mint).
// Outbound: Circulating token is taken from the source chain (lock or burn).